perfil/mandate/management/commands/load_deputy_info.py

```diff
@@ -23,17 +23,25 @@ class Command(ImportCsvCommand):
         'link',
         'parties_filiated',
         'term'
+        'congressperson_name',
+        'twitter_profile',
+        'secondary_twitter_profile',
+        'facebook_page',
     )
 
     def serialize(self, reader, total, progress_bar):
         for row in reader:
             person_id = self.cache.get(person_keys_birthdate(row))
             if person_id:
+                break
                 yield Politician(
```

Comment:
What is the point of this `break` followed by the `yield` indented at the same level? This is not clear for me. Maybe add a comment to clarify?

Reply:
this break should not be here... sorry